src/HttpClient/middlewares/cache.ts

```diff
@@ -11,23 +10,22 @@ const cacheableStatusCodes = [200, 203, 204, 206, 300, 301, 404, 405, 410, 414,
 export const cacheKey = (config: AxiosRequestConfig) => {
   const {baseURL = '', url = '', params, headers} = config
   const locale = headers[LOCALE_HEADER]
-  const fullURL = [baseURL, url].filter(str => str).join('/')
-  const urlObject = new URL(fullURL)
 
-  if (headers && headers.range) {
-    urlObject.searchParams.append(RANGE_HEADER_QS_KEY, headers.range)
-  }
+  const encodedBaseURL = baseURL.replace(/\//g, '\\')
+  const encodedURL = url.replace(/\//g, '\\')
+
+  let key = `${locale}--${encodedBaseURL}--${encodedURL}`
```

Comment:
I'd add a `?` in the end to indicate the base path has finished
```suggestion
  let key = `${locale}--${encodedBaseURL}--${encodedURL}?`
```